CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
 
 - Reversed Currencies Exchange to try resolving reverse of a currency pair
 
+### Fixed
+
+- Passing integer validation when chunk started with a dash
```

Comment:
As far as I can see you fixed quite a number of bugs with this PR, so maybe we should list them or write them in a more general way.